components/query/query-base.service.js

```diff
@@ -99,6 +99,12 @@ export default function (
         'pointermove',
         HsUtilsService.debounce(me.showPopUp, 500, false, me)
       );
+    }
+    if (me.customFeatureSelectionEnabled()) {
+      map.on(
+        'singleclick',
```

Comment:
Why not just subscribe to  this.HsEventBusService.vectorQueryFeatureSelection event?

Reply:
At first I started to use this for checking if customAction should be called at all, but to be fair I forgot about this event. Release 1.25 does not even have HsEventBus :D